File: benchmark/results/python-multilang/run-001/a-mcp/output/billing/services/tax_calculator.py
from billing.aggregates.invoice import Invoice

"""TaxCalculator service."""
class TaxCalculator:
    def create(self, input: dict) -> Invoice:
        raise NotImplementedError('not implemented')
    def find_by_id(self, id: str) -> Invoice | None:
        raise NotImplementedError('not implemented')
    def list(self, limit: int) -> list[Invoice]:
        raise NotImplementedError('not implemented')
    def delete(self, id: str) -> None:
        raise NotImplementedError('not implemented')

File: benchmark/results/java-multilang/run-005/b-baseline/output/com/metaengine/demo/billing/services/TaxCalculator.java
package com.metaengine.demo.billing.services;

import com.metaengine.demo.billing.aggregates.Invoice;
import java.util.List;

/** TaxCalculator service for the billing domain. */
public class TaxCalculator {

    /** Create a new Invoice. */
    public Invoice create(Invoice input) {
        throw new UnsupportedOperationException("not implemented");
    }

    /** Find an Invoice by id. May return null. */
    public Invoice findById(String id) {
        throw new UnsupportedOperationException("not implemented");
    }

    /** List invoices up to the given limit. */
    public List<Invoice> list(int limit) {
        throw new UnsupportedOperationException("not implemented");
    }

    /** Delete an Invoice by id. */
    public void delete(String id) {
        throw new UnsupportedOperationException("not implemented");
    }
}
